src/sidebar/components/Annotation/AnnotationShareControl.js

```diff
@@ -123,53 +142,69 @@ function AnnotationShareControl({
   );
 
   return (
-    <div className="AnnotationShareControl" ref={shareRef}>
+    <div className="relative" ref={shareRef}>
       <IconButton
         icon="share"
         title="Share"
         onClick={toggleSharePanel}
         expanded={isOpen}
       />
       {isOpen && (
-        <div className="annotation-share-panel">
-          <div className="annotation-share-panel__header">
-            <div className="annotation-share-panel__title">
+        <Card
+          classes={classnames(
+            // Prefer width 96 (24rem) but ensure that component isn't wider
+            // than 85vw
+            'w-96 max-w-[85vw]',
+            // Position this Card above its IconButton. Account for larger
+            // IconButtons in touch interfaces
+            'absolute bottom-8 right-1 touch:bottom-touch-minimum',
+            'space-y-2 p-2',
+            // Cards do not have a border in the clean theme. Turn it back on.
+            'theme-clean:border'
+          )}
+        >
+          <div className="flex items-center pb-2 border-b">
+            <h2 className="text-brand text-lg font-medium">
               Share this annotation
-            </div>
+            </h2>
+          </div>
+          <div
+            className={classnames(
+              // Slightly larger font size for touch devices to correspond with
+              // larger button and input sizes
+              'flex w-full text-sm touch:text-base'
+            )}
+          >
+            <TextInputWithButton>
+              <TextInput
+                aria-label="Use this URL to share this annotation"
+                type="text"
+                value={shareUri}
+                readOnly
+                inputRef={inputRef}
+              />
+              <IconButton
+                icon="copy"
+                title="Copy share link to clipboard"
+                onClick={copyShareLink}
+                variant="dark"
```

Comment:
This button looks a little narrow on desktop, although it is the usual 24px size we use for other icon buttons. Might make sense to make it larger given it is the primary action in this dialog.